v1.0/openapi3/info/index.yaml

```diff
@@ -1,6 +1,6 @@
 version: '1.0'
 title: 'Open API for FSP Interoperability (FSPIOP) (Implementation Friendly Version) Snippets'
-description: "Based on [API Definition version 1.0](https://github.com/mojaloop/mojaloop-specification/blob/develop/API%20Definition%20v1.0.pdf).\n**Note:** The API supports a maximum size of 65536 bytes (64 Kilobytes) in the HTTP header."
+description: "Based on [API Definition version 1.0](https://github.com/mojaloop/mojaloop-specification/blob/develop/API%20Definition%20v1.0.pdf).**Note:** The API supports a maximum size of 65536 bytes (64 Kilobytes) in the HTTP header."
```

Comment:
Same here, perhaps could be split into multiple lines? And would it be better to separate the note into a new line?

Reply:
done